app/src/main/java/ca/bc/gov/bchealth/ui/home/HomeViewModel.kt

```diff
@@ -195,6 +197,8 @@ class HomeViewModel @Inject constructor(
         if (bannerRequested.not()) {
             viewModelScope.launch {
                 try {
+                    mobileConfigRepository.getBaseUrl()
```

Comment:
after calling getBaseUrl() api, we are checking that is service up or down. and if its down we are showing service is down snackbar. please follow same flow here also.